src/main/java/com/ua/accommodation/dto/user/UserUpdateRoleDto.java

```diff
@@ -0,0 +1,15 @@
+package com.ua.accommodation.dto.user;
+
+import com.ua.accommodation.model.Role;
+import java.util.Set;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+@Data
+public class UserUpdateRoleDto {
+    private Long id;
```

Comment:
Why do you need id here? You accept user id as PathVariable and set of roles as RequestBody